viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if title and genre and rating:
+        movie['title'] = title
+        movie['genre'] = genre
+        movie['rating'] = rating
+        return movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    watched_movies = []
+    watched_movies.append(movie)
+    user_data["watched"] = watched_movies

```

Comment:
This has a bug that our tests don't catch.  If the `user_data["watched"]` list has content, this function will overwrite the list with a list containing only the new movie.
```suggestion
    user_data["watched"].append(movie)
```